### PR TITLE
Feature/fix 0 preview local worker test

### PR DESCRIPTION
**Pull request by pramaha**

### CODE REVIEW

**Comment by jason-shiao** [comment]
:star: PR built and available in a preview environment **fsa-streamotion-conductor-pr-87**

**Comment by jason-shiao** [comment]
:star: PR built and available in a preview environment **fsa-streamotion-conductor-pr-87**

File: charts/preview/values.yaml [hunk 1]
```diff
@@ -134,3 +137,13 @@ mysql:
...
     type: ClusterIP
     port: 3306
+
+bedrock-k8s-conductor-workers:
+  enabled: true
+  env:
+    SPRING_PROFILES_ACTIVE: local,demo
+    CONDUCTOR_SERVERAPIURI: "http://conductor-server/api/"
+  image:
+    repository: 053457794187.dkr.ecr.ap-southeast-2.amazonaws.com/fsa-streamotion/bedrock-k8s-conductor-workers
```
Comment (hoaphanFox) [inline charts/preview/values.yaml:147]: it'd start the app but I think it doesn't do anything aside from starting up.
also it'd create an interesting cyclic relationship between the 2 repos.

Comment (pramaha) [reply charts/preview/values.yaml:147]: agreed, this just helps validating that worflow-defs and task-defs are created without error. This is just for testing and won't be part of any actual PR

Comment (hoaphanFox) [reply charts/preview/values.yaml:147]: more direct way is in Jenkinsfile, add the step to clone that repo, then mvn verify ... (whatever the profile that start component test pointing to this conductor).

**Pull request closed without merging**